FILE-STORAGE.md

```diff
@@ -0,0 +1,53 @@
+# File Storage Configurations
+
+This project aims to support multiple file storage options which can be configured using the `STORAGE_TYPE` environment variable as shown in the settings file. Options are:
+
+- LOCAL (files are stored on the same filesystem as the hosting server)
+- GCP
+
+## Local Filesystem
+
+- Static files are stored in the `STATIC_ROOT` folder. Directly served by nginx.
+- User-uploaded media files are stored in the `MEDIA_ROOT` folder. Authorization is done by Django views and then an `X-Accel-Redirect` to nginx if approved.
+- When projects get published, if it is a public project, files get moved to a `published_projects` subdirectory within `STATIC_ROOT`. Otherwise they get moved into a `published_projects` subdirectory within `MEDIA_ROOT`.
+
+## GCS File Storage
```

Comment:
Could you add info regarding CORS configuration for the media bucket (required for direct uploads)